bitsandbytes/autograd/_functions.py

```diff
@@ -283,12 +276,17 @@ def tile_indices(self):
 
 
 class MatMul8bitLt(torch.autograd.Function):
-    # forward is the same, but we added the fallback for pre-turing GPUs
-    # backward is mostly the same, but adds one extra clause (see "elif state.CxB is not None")
-
     @staticmethod
-    def forward(ctx, A, B, out=None, bias=None, state=MatmulLtState):
-        using_igemmlt = supports_igemmlt(A.device) and not state.force_no_igemmlt
+    def forward(
+        ctx: torch.autograd.function.FunctionCtx,
+        A: torch.Tensor,
+        B: torch.Tensor,
+        out=None,
+        bias: Optional[torch.Tensor] = None,
+        state: MatmulLtState = None,
+    ):
+        state = state or MatmulLtState()
```

Comment:
This seemed like a bug/oversight to default `state=MatmulLtState` so I changed it here to use a new empty state.